tools/src/test/resources/1.5/valid-evidence-1.5.json

```diff
@@ -19,6 +19,40 @@
       ],
       "purl": "pkg:maven/com.google.code.findbugs/findbugs-project@3.0.0",
       "evidence": {
+        "identity": {
+          "field": "purl",
```

Comment:
I'm not sold on the idea that this "field" needs to even be there. The original idea was to be able to separate out individual evidence of group/name/version into individual pieces of evidence. But for some things, it actually doesn't make sense (like purl).

Reply:
Same would apply fo "field" in the XML and protobuf.